building milestones and releases does no longer work

https://ci.eclipse.org/xtext/job/xtext-monorepo-full-deploy-milestone/77/console

12:57:22  [WARNING] The POM for org.eclipse.tycho.extras:tycho-eclipserun-plugin:jar:5.0.2 is missing, no dependency information available
12:57:22  [ERROR] Plugin org.eclipse.tycho.extras:tycho-eclipserun-plugin:5.0.2 or one of its dependencies could not be resolved:
12:57:22  [ERROR] 	org.eclipse.tycho.extras:tycho-eclipserun-plugin:jar:5.0.2 was not found in https://repo.eclipse.org/content/repositories/cbi-releases/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of cbi-release has elapsed or updates are forced
12:57:22  [ERROR] 	org.eclipse.tycho.extras:tycho-eclipserun-plugin:jar:5.0.2 was not found in https://repo.eclipse.org/repository/maven-central/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of eclipse.maven.central.mirror has elapsed or updates are forced

--- a/org.eclipse.xtext.p2repository/pom.xml
+++ b/org.eclipse.xtext.p2repository/pom.xml
@@ -13,7 +13,7 @@
 	<properties>
 		<root-dir>${basedir}/..</root-dir>
 		<zipFinalName>${project.artifactId}-${qualifiedVersion}</zipFinalName>
-		<!-- Used for tycho-eclipserun-plugin -->
+		<!-- Used for tycho-eclipse-plugin -->
 		<eclipse-version>2023-03</eclipse-version>
 		<!-- For p2 mirroring -->
 		<p2.mirrorsURL>https://www.eclipse.org/downloads/download.php?format=xml&amp;file=/modeling/tmf/xtext/${remote-subdir}/${project.version}</p2.mirrorsURL>
@@ -206,8 +206,8 @@
 					</executions>
 				</plugin>
 				<plugin>
-					<groupId>org.eclipse.tycho.extras</groupId>
-					<artifactId>tycho-eclipserun-plugin</artifactId>
+					<groupId>org.eclipse.tycho</groupId>
+					<artifactId>tycho-eclipse-plugin</artifactId>
 					<configuration>
 						<repositories>
 							<repository>
@@ -399,8 +399,8 @@
 						</executions>
 					</plugin>
 					<plugin>
-						<groupId>org.eclipse.tycho.extras</groupId>
-						<artifactId>tycho-eclipserun-plugin</artifactId>
+						<groupId>org.eclipse.tycho</groupId>
+						<artifactId>tycho-eclipse-plugin</artifactId>
 						<executions>
 							<!-- add our new child repository -->
 							<execution>
@@ -469,8 +469,8 @@
 						</executions>
 					</plugin>
 					<plugin>
-						<groupId>org.eclipse.tycho.extras</groupId>
-						<artifactId>tycho-eclipserun-plugin</artifactId>
+						<groupId>org.eclipse.tycho</groupId>
+						<artifactId>tycho-eclipse-plugin</artifactId>
 						<executions>
 							<!-- add our new child repository -->
 							<execution>

--- a/pom.xml
+++ b/pom.xml
@@ -921,8 +921,8 @@
 					</configuration>
 				</plugin>
 				<plugin>
-					<groupId>org.eclipse.tycho.extras</groupId>
-					<artifactId>tycho-eclipserun-plugin</artifactId>
+					<groupId>org.eclipse.tycho</groupId>
+					<artifactId>tycho-eclipse-plugin</artifactId>
 					<version>${tycho-version}</version>
 				</plugin>
 				<plugin>

